### PR TITLE
pushing some cool code

### PR DESCRIPTION
**Pull request by troyleach**
https://trello.com/c/BeCeEakw/53-admin-view-specifications
# Cool Code
- I wrote some really cool code

### CODE REVIEW

File: app/controllers/beds_controller.rb [hunk 1]
```diff
@@ -3,6 +3,7 @@ class BedsController < ApplicationController
   
 
   def index
+    this is a cool change that brakes stuff
```
Comment (mrericdavis) [inline app/controllers/beds_controller.rb:6]: ```break ``` should be spelled like this

Comment (fredlcalhoun) [inline app/controllers/beds_controller.rb:6]: ```break```  should be spelled like this.

**Review by mrericdavis — COMMENTED** [review]
you did a really good job 🌮

**Review by fredlcalhoun — COMMENTED** [review]
you did a really good job 👍

**Pull request closed without merging**